spec/metrics-spec.js

```diff
@@ -121,6 +121,26 @@ describe('Metrics', async () => {
       let url = Reporter.request.mostRecentCall.args[0]
       expect(url).toContain('aiid=stable')
     })
+
+    it('reports an arbitrary release channel', async () => {
+      spyOn(atom, 'getVersion').andReturn('1.0.2-sushi1')
+
+      await atom.packages.activatePackage('metrics')
+      await conditionPromise(() => Reporter.request.callCount > 0)
+
+      let url = Reporter.request.mostRecentCall.args[0]
+      expect(url).toContain('aiid=sushi')
+    })
+
+    it('reports an unknown release channel', async () => {
```

Comment:
'unknown' may not be the best name for this because it should indicate an Atom version that doesn't conform to the expected pattern.  Any thoughts?

Reply:
Maybe "nonstandard" or "unrecognized"?

Reply:
:+1: 'unrecognized'